test/karma.conf.js

```diff
@@ -85,26 +95,12 @@ module.exports = function(config) {
 
     // define browsers
     customLaunchers: {
-      bs_firefox_85: {
-        base: 'BrowserStack',
-        browser: 'Firefox',
-        browser_version: '85.0',
-        os: 'OS X',
-        os_version: 'Big Sur'
-      },
-      bs_chrome_88: {
-        base: 'BrowserStack',
-        browser: 'Chrome',
-        browser_version: '88.0',
-        os: 'OS X',
-        os_version: 'Big Sur'
-      },
-      bs_safari_14: {
+      bs_safari_15: { // Webkit and Safari can differ in behavior
         base: 'BrowserStack',
         browser: 'Safari',
-        browser_version: '14.0',
+        browser_version: 'latest',
```

Comment:
This should say `15.0`, no? Or alternatively the name of the browser should be `bs_safari_latest`